src/pages/FocusPage.tsx

```diff
@@ -315,12 +318,14 @@ export const FocusPage = ({
         />
       )}
       {showEstimatedCasesDivGrid && country === 'Switzerland' && (
-        <SwitzerlandEstimatedCasesDivisionModal
-          variantSampleSet={variantSampleSet}
-          wholeSampleSet={wholeSampleSet}
-          show={showEstimatedCasesDivGrid}
-          handleClose={() => setShowEstimatedCasesDivGrid(false)}
-        />
+        <QueryClientProvider client={queryClient}>
```

Comment:
We have to enclose the functional component with this `<QueryClientProvider>` tag which I think can be placed at a even higher level like on the whole page component or App component